src/host/usbh.c

```diff
@@ -860,7 +863,7 @@ bool usbh_edpt_xfer_with_callback(uint8_t dev_addr, uint8_t ep_addr, uint8_t * b
     // HCD error, mark endpoint as ready to allow next transfer
     ep_state->busy    = 0;
     ep_state->claimed = 0;
-    TU_LOG1("Failed\r\n");
+    TU_LOG_USBH("Failed\r\n");
```

Comment:
keep this as LOG1, since failed is always print out with LOG LEVEL = 1

Reply:
done